whispir/openapi
# Added OpenAPI Code Generator module for Node and Java

Here are the following steps to achieve to test Node SDK that is auto-generated:
1.  cd into `whispir-sdk-codegen`
2. run `yarn generate` -> this will auto generate SDK based on `openapitools.json` config
3. cd into `whispir-sdk-node` (the generated SDK) 
4. run `yarn Install` & `yarn build`
5. cd into `whispir-test`
6. make sure you run `yarn install`
7. run `yarn test`

you would be able to see this response as part of the console 
![image](https://user-images.githubusercontent.com/99852493/187621795-1fb02082-3338-41b2-81aa-2fd1e52f61b5.png)
